_posts/2025-05-31-run-pr.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Try out any JabRef pull request
+---
+
+Thanks to the strong ecosystem of Java, one can easily try out (nearly) any JabRef pull request.
```

Comment:
Some more context, for example 
you want to try out a feature in development or fix and give feedback

Reply:
Done.